### PR TITLE
Add FXIOS-8590 - Swiftlint autocorrect Focus iOS

### PR DESCRIPTION
**Pull request by adudenamedruby**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-8590)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/19081)

## :bulb: Description
Run `swiftlint --fix` for rules that can be autocorrected

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

**Review by lmarceau — APPROVED** [review]

**Comment by isabelrios** [comment]
So that swiftlint for focus run, we have to either:
- Enable the swiftlint in the `focus_configure_build` step: https://github.com/mozilla-mobile/firefox-ios/blob/d57bd6e4ff2ff5562e35b5c34a6cbea4f58b7787/bitrise.yml#L1589
- Remove the focus-ios path from the .swiftlint file in root dir: https://github.com/mozilla-mobile/firefox-ios/blob/d57bd6e4ff2ff5562e35b5c34a6cbea4f58b7787/.swiftlint.yml#L136

On top of that.. I'm not really sure which swiftlint file is going to be taken when running focus.. you may know though

**Comment by isabelrios** [comment]
Bitrise build failed due to a lint error when building for focus:

> ▸ Running script 'Run Swiftlint'
>     'unneeded_override' is not a valid rule identifier
> ❌  /Users/vagrant/git/firefox-ios/Storage/ThirdParty/SwiftData.swift:321:12: Yoda Condition Violation: The constant literal should be placed on the right-hand side of the comparison operator (yoda_condition)
> Linting 'firefox-ios/firefox-ios-tests/Tests/UITests/DomainAutocompleteTests.swift' (177/1760)
>                   ^
> ** TEST BUILD FAILED **
>

File: focus-ios/.swiftlint.yml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  # so must be excluded due to file_header rule
+  - firefox-ios/Package.swift
+  - BrowserKit/Package.swift
+  - BrowserKit/.build/
+  - firefox-ios/Client/ContentBlocker/ContentBlockerGenerator/Package.swift
+  - Package.swift
+  - firefox-ios/Build/Intermediates.noindex/Client.build/Fennec-iphoneos/WidgetKitExtension.build/DerivedSources/IntentDefinitionGenerated/WidgetIntents/*
+  # Temporary for Focus
+  - Blockzilla/Generated/
+  - /Users/vagrant/git/DerivedData/
```
Comment (isabelrios) [inline focus-ios/.swiftlint.yml:137]: Looks like we need to add also:
`firefox-ios/fastlane/`

https://app.bitrise.io/build/55106801-0d2a-4cc4-a26c-ac2b87e7e6d6

Comment (adudenamedruby) [reply focus-ios/.swiftlint.yml:137]: hmmmm. But that's already in line 115. Maybe I need to ignore `users/vagrant/git/firefox-ios/fastlane`?

Comment (isabelrios) [reply focus-ios/.swiftlint.yml:137]: but, if I'm not wrong that's in the root swiftlint not in the one we are using here..

**Comment by isabelrios** [comment]
@adudenamedruby this is green, should we go ahead and land it?

**Comment by adudenamedruby** [comment]
Ah, dang. Now there's conflicts. Sorry. I was sick all last week and didn't get a chance to merge this. Ok. I'll fix, repush, and merge. :D

**Pull request merged**